iNaturalist/src/main/res/layout/default_licenses.xml

```diff
@@ -1,117 +1,122 @@
+                </LinearLayout>
+            </RelativeLayout>
+        </LinearLayout>
 
-    <LinearLayout
-        android:id="@+id/change_licenses_for_existing_observations"
-        android:layout_width="match_parent"
-        android:layout_height="wrap_content"
-        android:minHeight="50dp"
-        android:gravity="center_vertical"
-        android:paddingLeft="26dp"
-        android:paddingRight="26dp"
-        android:clipToPadding="false">
-
-        <RelativeLayout
-            android:layout_width="wrap_content"
+
+        <View
+            android:layout_width="match_parent"
+            android:layout_height="1dp"
+            android:layout_marginTop="5dp"
+            android:layout_marginBottom="5dp"
+            android:background="#D5D5D5" />
+
+        <LinearLayout
+            android:id="@+id/change_licenses_for_existing_observations"
+            android:layout_width="match_parent"
             android:layout_height="wrap_content"
-            android:layout_weight="1"
-            android:paddingTop="16dp"
-            android:paddingBottom="16dp">
+            android:clipToPadding="false"
+            android:gravity="center_vertical"
+            android:minHeight="50dp"
+            android:paddingLeft="26dp"
+            android:paddingRight="26dp">
 
-            <TextView
+            <RelativeLayout
                 android:layout_width="wrap_content"
                 android:layout_height="wrap_content"
-                android:text="@string/change_licenses_for_existing_observations"
-                android:textColor="#000000"
-                android:textStyle="bold"
-                android:textSize="17sp"
-                android:ellipsize="marquee" />
-
-        </RelativeLayout>
-
-        <ImageView
-            android:gravity="end|center_vertical"
-            android:layout_width="30dp"
-            android:layout_marginLeft="10dp"
-            android:layout_height="wrap_content"
-            android:src="@drawable/ic_keyboard_arrow_right_black_48dp"
-            android:tint="#898b8c"
-            android:importantForAccessibility="no"
-            />
+                android:layout_weight="1"
+                android:paddingTop="16dp"
+                android:paddingBottom="16dp">
 
-    </LinearLayout>
+                <TextView
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:ellipsize="marquee"
+                    android:text="@string/change_licenses_for_existing_observations"
+                    android:textColor="#000000"
+                    android:textSize="17sp"
+                    android:textStyle="bold" />
+
+            </RelativeLayout>
 
-</LinearLayout>
\ No newline at end of file
+            <ImageView
+                android:layout_width="30dp"
+                android:layout_height="wrap_content"
+                android:layout_marginLeft="10dp"
```

Comment:
And here it should be `android:layout_marginStart`, again just to make sure this doesn't remain as `Left` when we merge the RTL branch.